src/argparse.cpp

```diff
@@ -82,6 +83,8 @@ void parseArguments(int argc, char **argv, std::vector<std::string>& slot_list,
                       << " DEBUG"
 #endif
                       << "\n";
+            if (!commit_hash.empty())
+               std::cout << "From source hash: " << commit_hash << "\n";
```

Comment:
You're right, there's no release process defined, that's why I said 'Food for thought' :-)